src/devices/Nmea0183/tests/GeodeticFunctionTests.cs

```diff
@@ -293,8 +293,8 @@ public void RouteCalculationInverse()
         {
             GeographicPosition start = new GeographicPosition(0, 1, 0);
             GeographicPosition end = new GeographicPosition(0, 2, 0);
-            var points = GreatCircle.CalculateRoute(start, end, 1000);
-            Assert.Equal(112, points.Count);
+            var points = GreatCircle.CalculateRoute(start, end, Length.FromMeters(1000));
+            Assert.Equal(113, points.Count);
```

Comment:
why did this test need adjustment? was it incorrect before?

Reply:
IIRC, some change caused the first or last point to be included in the path as well, which is correct. So this change reflects a fix in the implementation.